src/components/layout/Navbar/NavBarLink/styles.css

```diff
@@ -6,4 +6,5 @@
   font-size: 16px;
   font-family: 'Rubik-Bold';
   text-decoration: none;
+  opacity: 50%;
```

Comment:
Should be 100% by default according to figma unless a menu is open

Reply:
turned it back to 50%, will add some logic later for when the menu is open